.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```
@@ -223,7 +223,9 @@ stages:
       - job: deploy_pods
         condition: and( and( not(canceled()), not(failed()) ), or( contains(variables.CONTROL_SCENARIO, 'scaleTest') , contains(variables.CONTROL_SCENARIO, 'all') ) )
         displayName: "Scale Test"
-        dependsOn: deploy_cilium_components
+        dependsOn: 
+          - deploy_cilium_components
+          - deploy_cns_and_ipam

```

nit: unless `deploy_cilium_components` is setting variables within AZP you do not need to have it set it within `dependsOn` as `deploy_cns_and_ipam` already requires it.

```suggestion
        dependsOn: deploy_cns_and_ipam

```

---

going to keep this for now, will have to check on the cilium templates to confirm before I remove it